api/src/main/java/org/openmrs/module/fhir2/FhirTaskInput.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.fhir2;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import org.hl7.fhir.r4.model.Task;
+
+/**
+ * FHIR Task.input - https://www.hl7.org/fhir/task-definitions.html#Task.input
+ */
+@Data(staticConstructor = "of")
+@NoArgsConstructor
+@EqualsAndHashCode(callSuper = true)
+@Entity
+@Table(name = "fhir_task_input")
+public class FhirTaskInput extends FhirTaskParam {
+	
+	private static final long serialVersionUID = 1L;
+	
+	@Id
+	@GeneratedValue(strategy = GenerationType.AUTO)
+	@Column(name = "task_input_id")
+	private Integer id;
+	
+	/**
+	 * FhirTaskInput.type: A code or description indicating how the input is intended to be used as part
```

Comment:
I get what these are doing, but we should probably either 1) add a function here or 2) just add a Javadoc on the `FhirTaskParam` that doesn't explicitly reference input or output.